### ISSUE TITLE
Blood Moon Feedback Issue

### ISSUE THREAD

**Issue by Shadowmech88** [d3athrow/vgstation13#16397]
@MadmanMartian

**Comment by Blithering** [comment]
Pasted from the other issue. Didn't get to see the whole round as I joined late.

The main problem appeared to be that it instantly spawns mobs with no prior warning, leaving the crew flatfooted and dead within the first few moments. A global message when the button is pressed, followed by a few minutes prep time could really make it more enjoyable.

Another problem is that with the APCs all breaking, anyone without easy access to a crowbar gets stuck in their department even if they can kill off the mobs that spawn there. Perhaps a drop off of survival equipment like meteor mode has would help with that?

The other thing that might help would be clear goals, perhaps a way to defeat the event so that the crew can return to normality somewhat or at least call the shuttle to evacuate the survivors.

**Comment by Pathid** [comment]
`var/choice = input("Are you sure you want to wake up the space indian burial ground?. Misuse of this could result in removal of flags or hilarity.") in list("Get our spook on", "Cancel")`

Might want to add to that, "Seriously, only generously prepared crew can hope to survive this."

**Comment by Killette2** [comment]
It's mean. It's gay. All the skeletons are dummyheads.

Power goes out really fast, it's hard to play if you didn't find a tool first which I couldn't find in time :frowning_face: . One zombie I hit 4 times with a sword and it didn't die while it's hits did about 40 brute to me, I had no idea it was this strong without something to make it more distinct. Enemies were really clumped together and I really got overwhelmed. Even if you do end up killing them off yourself, the damage they have caused from their massive spawning makes recovering less fun than the most dull dwarf fortress round.

**Comment by Irradiation** [comment]
-The power on the station goes out completely WITHOUT WARNING in about 10 to 30 seconds. The power outage causes airlocks to be unopenable unless you have access to a Crowbar or any other airlock opener.
The problem with this is that people don't get a warning and more importantly, not a whole lot of jobs actually get access to a crowbar fast. In order to remediate this issue, players should be warned about the impending power outage in advance, preferably 1-3 minutes before it occurs. This should give players enough time to equip themselves with crowbars and other survival gear.

-The previously mentionned power outage is followed by a rather large and condensed spawn of whatever variation of zombie-type mobs (Cyber Horrors, Glowing Ones, Zombies, Leatherman(?). The station, and more importantly the separate departments, can get flooded with HIGHLY ROBUST mobs in large quantities.
The issue in this situation is that the players are excessively outnumbered and outrobusted by a huge spawn of robust mobs. The mob spawns should be lowered across the board and should be spawned in a less condensed fashion in order to allow breathing room on the station. It would give players a mildly better chance to fight back, or at least, run away and hide since that seems to be the idea of this gamemode.

-Following the lack of warning and the power outage, there is no apparent objective for the players. Unless they have prior knowledge of the gamemode, players won't know what is going down and will be promptly killed (and converted into zombies?). Of course, informations will be spread across the playerbase, but simply "Surviving" without any explanations, side-objectives or shuttle call possibility is boring and a let-down.
To remediate this, there should be a message telling the players that shit is going to hit the fan, how long they need to survive and what are their  options? If the power is down, how do they call the shuttle? In the round we played, Death Squads and ERTs both got called and were doing their usual KILL EVERYTHING and SAVE PEOPLE AND KILL MOBS respectively. Maybe the players could escape on the ERT shuttle?

**Comment by Pathid** [comment]
It's not accurate to call it a gamemode or compare it to meteor. This thing by all appearances is supposed to be an endgame event on par with Nar-sie getting summoned or a supermatter cascade, not something admins should be pressing five minutes into the round. That said, even those have inbuilt "win" conditions: Hell Rising calls the shuttle and the Cascade has the rift you can escape through. Blood Moon doesn't even end the round on its own.

**Comment by AdminbusVictim** [comment]
Alright, I'm making a new account to throw my hat into the ring since I forgot my old GitHub account.

This really didn't feel like a worthwhile time. I spawned in as QM before watching power go completely black, the station went red, and skeletons spawned en masse around me. I didn't really have any weapons, and some of them were using the equivalent of a flamethrower. I think I was overwhelmed before I could even think about taking one of the 5 that spawn around our lobby down. To add to that, it's not like I could kite them or run away from them since power got cut.

I'd recommend offering the crew preparation time and clear warning on what is coming so that the event can have the potential to be fun for everyone involved. I died right as the event happened and watched as 4 players got to live out their power fantasy as death squad members which is probably a mistake for an adminbus since so many were taken out of the round so fast.

**Comment by MadmanMartian** [comment]
This is some really good feedback. I'll get onto it as soon as I am able.

10~ minute warning, with the stars going red first, then 2 minutes to doom the APCs die out?

It was really not supposed to be a roundstart thing, but more of a round ender where the crew has to stick together. I'll add the shuttle call and such.

@AdminbusVictim Cargo is the second most dangerous place to be, first being medical, third being the kitchen. But I definitely see what you're saying with the feedback.

**Comment by AdminbusVictim** [comment]
Just coming off the latest test of it, it seems like jackal pyromaniacs are much more lethal than other undead.

I guess since I don't have the full picture of what's where and spawn areas seem intentional, this might actually be fine. If this type of round becomes more standard in expectations, I think there can be a lot of emergent fun behind a crew collectively deciding where to hold out for a period of time before a shuttle call and a mass migration to a shuttle. But that might raise another point - should the hallways be much more crowded with mobs in an effort to present more of a straightforward challenge to people trying to charge to escape?

All-in-all, the latest run on deff was pretty decent and I have a better appreciation of what you guys were setting out to do after seeing what the crew is able to do with legitimate setup time.

**Comment by Hugh-G-Rection** [comment]
It should be pointed out that "Legitimate setup time" included a good deal of help from admins and a lawless MoMMi(me). Most everyone had high tier weapons and high frequency blades, and there were I think three combat mechs. As a bussed MoMMi I helped wall us off, replaced all the APCs so we had medbay power, rebuilt cloning and built about 8 turrets. Somehow even with all this only one or two people made it out alive. 
Not giving any criticism or praise one way or the other, just want to clarify on the success of that round. It was fun, but it still wasn't just a button that could be hit and walked away from.

**Comment by Blithering** [comment]
Perhaps there should be an option to give the crew helper packages, something that gives a weapon and some armour like summon swords but without the added traitors.

**Comment by MadmanMartian** [comment]
That's certainly feasible, what would the supply packs be?

suddenly appearing chests, or things appearing in the vicinity of a player?

To what degree of weaponry? Swords? Knives? Laser pistols? 

Medical supplies? Brute packs or trauma bandages?

I don't want to undersupply, but I don't want to oversupply. Maybe have it be luck of the draw means that person's more specialized at one thing than others might be (Who's going to go first, the guy with the sawed off full of buckshot, or the guy with the plasma pistol but has a revi-vial ready to go should somebody die.)

**Comment by Irradiation** [comment]
In my experience of fighting these mobs, melee fighting is generally suicide, as a few hits in melee will be enough to damage and slow you down enough for the mobs to quickly put you into crits.
As such, you could give a place a few strong melee weapons in the supply packs, preferably with a Riot/Energy shield. The bulk of the weaponry should be ranged weapons.

Regarding the damage type, the mobs seem to inflict heavy bleeding and brute damage, so anything short of a trauma bandage/kit will definitely not be enough. Putting some Inaprovaline and Dexalin (Plus) would be gravy to prevent crit symptoms.

As for item spawns, I believe that they should spawn in crates in specific areas, such as departments, although that could be changed ,seeing as departments can be bumrushed by mobs pretty fast.

As a final note, can zombies even be tripped? I got dropped down because lolZAS no magboots while most of the mobs could chase me down with no issues. When I finally got magboots and ufll health, they were still faster than me with no ZAS-related speed drawbacks.

**Comment by Shadowmech88** [comment]
What about combat supplies based on the departments? Chef gets a cleaver and some healing food, Cargo gets a Ripley with a drill, Toxins gets some pre-loaded TTVs and a blast cannon, et cetera.

**Comment by Shadowmech88** [comment]
@Irradiation 
I don't believe any simple animals, zombies included, can be stunned by any means.

**Comment by Hugh-G-Rection** [comment]
Love that idea, Shadow. The *fun* part of any zombie invasion is everyone fighting back with what they have at hand.
Which reminds me of something I haven't suggested in years, we should have better ghetto armor options. Picturing the chef and botanist wielding cleavers and novaflowers wearing tableparts held together with vines and reinforced bucket helmets.

**Comment by Blithering** [comment]
I would advise against guns, limited ammo and lack of power means any gun will end up useless, while a hatchet will be a hatchet the whole round.

**Comment by Aurx** [comment]
Guns give you the ability to engage without taking damage, or from behind fortifications.  
They also give you that wonderful "low on ammo" panic.  

I say give them some guns, but not enough ammo to use them for long.

**Comment by Hugh-G-Rection** [comment]
Lack of power isn't permanent, like I mentioned above I replaced APCs and got power up and running again in medbay, chem, science and cloning in about 20 minutes.

**Comment by Killette2** [comment]
How did you replace the APCs? When I tried to fiddle with one it said the interface was broken, no matter which tool or ID I tried to use on it.

However I WAS able to access an APC after the cover was unlocked, but due to the interface being broken the only way I was able to access it was a grimlin nearby was pushing buttons for me and disabled the cover lock that I wasn't able to switch allowing me to crowbar it open and replace the cell.
